### ISSUE TITLE
Accessibility 

### ISSUE THREAD

**Issue by Brycce** [VikeLabs/lecshare-main#7]
Our product is being built for people with accessibility challenges, research some of the ways that we can use a11y patterns to make sure we're accommodating for these people: https://a11yproject.com/patterns/

Check in with Bryce once you get started. 

Deliverables:
- A google doc outlining what we should be doing to build for people with accessibility issues.
